### DISCUSSION TITLE
Support/guidance on closed captioning

### DISCUSSION THREAD

**Discussion by jywarren** [mozilla/hubs#5224] [General Questions & Discussion] [answered]
I've been reading some great material on accessible VR spaces like https://equalentry.com/constructing-an-accessible-social-vr-environment/#h.qnphrakuswru - and wanted to know if there is, or could be, some more how-to guidance on setting up captioning. Thank you!

**Comment by matthewbcool** [comment, marked as answer]
Generally- screen sharing and making use of [mediaframes](https://hubs.mozilla.com/docs/spoke-adding-scene-content.html#media-frames). There is a tool called [webcaptioner](https://webcaptioner.com/) which you can use with OBS to generate live captions in-room. Converting this issue to a discussion. Welcome to discuss further in discord as well https://discord.gg/sBMqSjCndj

an in-depth guide on setting up OBS with webcaptioner or preferably human real-time transcription would be welcome.

**Reply by jywarren** [reply]
Thank you, I will research more for upcoming workshops I'm doing and if I set it up I'll document. But it may take me some time.

**Reply by jywarren** [reply]
Just updating to say that I'm thinking of trying to run captions in a parallel service, because I'll be giving a tour of a space (https://unterbahn.com/chinatown) where we'll be on the move. I wonder if there is a way to have webcaptioner appear as part of the comment stream using webhooks, or in subtitles above the user speaking? (supertitles? lol)

Thanks, just taking notes but if anyone has insights, thanks in advance!
